rosdep/python.yaml

```diff
@@ -264,6 +264,13 @@ pynput-pip:
   ubuntu:
     pip:
       packages: [pynput]
+pyosmium:
+  debian: [python-pyosmium]
+  fedora: [python3-pyosmium]
+  gentoo:
+    pip:
```

Comment:
Hm, this is a bit weird.  There are definitely instances of keys where we have a mixed package/pip key, but as far as I can tell those are all for osx.  I guess it is possible for gentoo, but I think this would cause consumers of the key to fail to build binary packages on gentoo.  On the other hand, maybe this isn't a problem; @tfoote @nuclearsandwich , any insights here?

Reply:
I did not find a matching native package in gentoo [searching Portage](https://packages.gentoo.org/). So I figured it was best to fallback on using the equivalent pip package.

Reply:
Would it be better to just remove the gentoo key?

Reply:
I'd suggest removing the key. I believe that gentoo generally doesn't like mixing pip onto the system. Unless you're using it a non-existant key is clearer messaging that it's not expected to work versus the key that's never been tested might suprise people if it doesn't work.